From 4f41e4c4ef9ae295b3cdb202e64a9710640f467f Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Thu, 9 Oct 2014 14:47:24 -0700
Subject: [PATCH 4932/5965] [PD #97597] [OSD] Fix compilation error in macro
 define.

Macro definition should not have semi-colon included, remove all
semi-colon in the macro defined in OSD RDMA driver.

Change-Id: I2f67bbb7e76db042f631703b9999d1c4ba38d67f
---
 drivers/amlogic/display/osd/osd_rdma.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/amlogic/display/osd/osd_rdma.c b/drivers/amlogic/display/osd/osd_rdma.c
index 75216d177caf..9b542a5a7d82 100755
--- a/drivers/amlogic/display/osd/osd_rdma.c
+++ b/drivers/amlogic/display/osd/osd_rdma.c
@@ -43,16 +43,16 @@ static int  osd_rdma_init(void);
 #define Wr(adr,val) WRITE_VCBUS_REG(adr, val)
 #define Rd(adr)    READ_VCBUS_REG(adr)
 #define Wr_reg_bits(adr, val, start, len)  WRITE_VCBUS_REG_BITS(adr, val, start, len)
-#define Wr_set_reg_bits_mask(adr, _mask)	 SET_VCBUS_REG_MASK(adr, _mask);
-#define Wr_clr_reg_bits_mask(adr, _mask)	 CLEAR_VCBUS_REG_MASK(adr, _mask);
-#define REGS_RECONFIG(reg) VCBUS_REG_ADDR(reg);
+#define Wr_set_reg_bits_mask(adr, _mask)	 SET_VCBUS_REG_MASK(adr, _mask)
+#define Wr_clr_reg_bits_mask(adr, _mask)	 CLEAR_VCBUS_REG_MASK(adr, _mask)
+#define REGS_RECONFIG(reg) VCBUS_REG_ADDR(reg)
 #else
 #define Wr(adr,val) WRITE_MPEG_REG(adr, val)
 #define Rd(adr)    READ_MPEG_REG(adr)
 #define Wr_reg_bits(adr, val, start, len)  WRITE_MPEG_REG_BITS(adr, val, start, len)
-#define Wr_set_reg_bits_mask(adr, _mask)	 SET_MPEG_REG_MASK(adr, _mask);
-#define Wr_clr_reg_bits_mask(adr, _mask)	 CLEAR_MPEG_REG_MASK(adr, _mask);
-#define REGS_RECONFIG(reg) CBUS_REG_ADDR(reg);
+#define Wr_set_reg_bits_mask(adr, _mask)	 SET_MPEG_REG_MASK(adr, _mask)
+#define Wr_clr_reg_bits_mask(adr, _mask)	 CLEAR_MPEG_REG_MASK(adr, _mask)
+#define REGS_RECONFIG(reg) CBUS_REG_ADDR(reg)
 #endif
 
 static int  update_table_item(u32 addr,u32 val)
-- 
2.19.0

